hera_mc/qm.py

```diff
@@ -0,0 +1,175 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Handling quality metrics of HERA data.
+
+"""
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+from astropy.time import Time
+from astropy.coordinates import EarthLocation
+from math import floor
+from sqlalchemy import (Column, Integer, BigInteger, Float, ForeignKey,
+                        String, ForeignKeyConstraint)
+from sqlalchemy.ext.hybrid import hybrid_property
+
+from hera_mc import geo_handling
+from . import MCDeclarativeBase, DEFAULT_GPS_TOL, DEFAULT_DAY_TOL, DEFAULT_HOUR_TOL
+
+
+class ant_metrics(MCDeclarativeBase):
+    """
+    Definition of ant_metrics table.
+
+    obsid:      observation identification number, generally equal to the floor
+                of the start time in gps seconds (long integer)
+    ant:        Antenna number (int >= 0)
+    pol:        Polarization ('x' or 'y')
```

Comment:
Thought about this too. Right now 'x' and 'y' are how things are treated in hera_qm, and I haven't seen much progress toward 'N' and 'E', other than the 'x_orientation' parameter in pyuvdata. But I can be persuaded.